subprojects/gradle/deeplink-generator/src/main/kotlin/com/avito/deeplink_generator/model/Deeplink.kt

```diff
@@ -0,0 +1,34 @@
+package com.avito.deeplink_generator.model
+
+import java.io.Serializable
+
+public class Deeplink(
```

Comment:
Will this model be used in clients' applications code?

Reply:
It may be used in client build scripts, as a type safe way to store public links information.